src/thor/timedep_forward.cc

```diff
@@ -212,10 +279,16 @@ inline bool TimeDepForward::ExpandForwardInner(GraphReader& graphreader,
   // less cost the predecessor is updated and the sort cost is decremented
   // by the difference in real cost (A* heuristic doesn't change)
   if (meta.edge_status->set() == EdgeSet::kTemporary) {
-    EdgeLabel& lab = edgelabels_[meta.edge_status->index()];
+    // TODO(danpat): can we slices down to EdgeLabel here safely?
+    EdgeLabel& lab =
+        FORWARD ? edgelabels_[meta.edge_status->index()] : edgelabels_[meta.edge_status->index()];
```

Comment:
Similar to above - previously had different EdgeLabel/BDEdgeLabel for forward/reverse - left this in as a temporary reminder until I confirm it's OK to use BDEdgeLabel in all cases.